### PR TITLE
PIL-975 #Revised design for payments page

### PR DESCRIPTION
**Pull request by javed-iqbal1**

### CODE REVIEW

File: app/views/MakeAPaymentDashboardView.scala.html [hunk 1]
```diff
@@ -75,17 +75,13 @@
 
 
         <h1 class="govuk-heading-l">@messages("payment.dashboard.header")</h1>
-        @govukInsetText(InsetText(content = Text(messages("payment.dashboard.insetText"))))
-        <p class="govuk-body govuk-!-margin-bottom-5" >@messages("payment.dashboard.message1")</p>
         <h2 class="govuk-heading-m">@messages("payment.dashboard.header2")</h2>
```
Comment (jassalrichy) [inline app/views/MakeAPaymentDashboardView.scala.html:78]: Can we use the gds.components.*

File: conf/messages.en [hunk 2]
```diff
@@ -173,24 +173,27 @@ grs.registration.error.b33 =  and try again using different details if you think
 payment.dashboard.title= Pay your Pillar 2 top-up taxes
 payment.dashboard.header= Pay your Pillar 2 top-up taxes
 payment.dashboard.insetText= You have no payments due.
-payment.dashboard.message1= You can make a voluntary payment based on self-assessed liability.
+payment.dashboard.message1= Payments can only be made by bank transfer.
 payment.dashboard.header2= How to make a payment
-payment.dashboard.message2= Payments can be made by bank transfer.
-payment.dashboard.message3= To make a bank transfer payment, you can visit your bank’s website, use their mobile app, or go to a branch and use HMRC’s banking details.
+payment.dashboard.message2= To make a bank transfer, you can visit your bank''s website, use their mobile app, call the number at the back of your card or go to a branch. You will need to provide your bank with HMRC''s bank details.
```
Comment (jassalrichy) [inline conf/messages.en:178]: Incorrect apostrophe: ... bank''s website ...

File: conf/messages.en [hunk 3]
```diff
@@ -173,24 +173,27 @@ grs.registration.error.b33 =  and try again using different details if you think
...
 payment.dashboard.table.row_11= 08 32 10
 payment.dashboard.table.row_21= 12001020
 payment.dashboard.table.row_31= HMRC Shipley
 payment.dashboard.table.row_12= Sort code
 payment.dashboard.table.row_22= Account number
 payment.dashboard.table.row_32= Account name
-payment.dashboard.table.caption= HMRC’s bank details
+payment.dashboard.table.caption= HMRC''s bank details for UK payments
+
+payment.dashboard.table.outsideUK.caption= HMRC''s bank details for payments outside the UK
```
Comment (jassalrichy) [inline conf/messages.en:196]: Incorrect apostrophe:
... HMRC''s bank ...

File: conf/messages.en [hunk 4]
```diff
@@ -173,24 +173,27 @@ grs.registration.error.b33 =  and try again using different details if you think
...
+payment.dashboard.warning.message = You must use {0} as your payment reference.  We need this to match your payment.
 payment.dashboard.outsideUK-summary= Make a payment from outside the UK
 payment.dashboard.table.row_11= 08 32 10
 payment.dashboard.table.row_21= 12001020
 payment.dashboard.table.row_31= HMRC Shipley
 payment.dashboard.table.row_12= Sort code
 payment.dashboard.table.row_22= Account number
 payment.dashboard.table.row_32= Account name
-payment.dashboard.table.caption= HMRC’s bank details
+payment.dashboard.table.caption= HMRC''s bank details for UK payments
```
Comment (jassalrichy) [inline conf/messages.en:194]: Incorrect apostrophe:
... HMRC''s bank ...

File: conf/messages.en [hunk 5]
```diff
@@ -173,24 +173,27 @@ grs.registration.error.b33 =  and try again using different details if you think
...
+payment.dashboard.warning.message = You must use {0} as your payment reference. We need this to match your payment.
 payment.dashboard.outsideUK-summary= Make a payment from outside the UK
 payment.dashboard.table.row_11= 08 32 10
 payment.dashboard.table.row_21= 12001020
 payment.dashboard.table.row_31= HMRC Shipley
 payment.dashboard.table.row_12= Sort code
 payment.dashboard.table.row_22= Account number
 payment.dashboard.table.row_32= Account name
-payment.dashboard.table.caption= HMRC’s bank details
+payment.dashboard.table.caption= HMRC''s bank details for UK payments
```
Comment (jassalrichy) [inline conf/messages.en:194]: Incorrect apostrophe on line 194.

File: test/views/MakeAPaymentDashboardViewSpec.scala [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+
+    "have warning text" in {
+      val wText = makePaymentDashboardView.getElementsByClass("govuk-warning-text__assistive")
+      wText.text must include("You must use 12345678 as your payment reference. We need this to match your payment.")
+
+    }
+
+    "table caption for account details" in {
+      val tCaption = makePaymentDashboardView.getElementsByClass("govuk-table__caption")
+      tCaption.get(0).text must include("HMRC's bank details for UK payments")
```
Comment (jassalrichy) [inline test/views/MakeAPaymentDashboardViewSpec.scala:56]: Incorrect apostrophe on line 56

File: conf/messages.en [hunk 7]
```diff
@@ -173,24 +173,27 @@ grs.registration.error.b33 =  and try again using different details if you think
 payment.dashboard.title= Pay your Pillar 2 top-up taxes
 payment.dashboard.header= Pay your Pillar 2 top-up taxes
 payment.dashboard.insetText= You have no payments due.
-payment.dashboard.message1= You can make a voluntary payment based on self-assessed liability.
+payment.dashboard.message1= Payments can only be made by bank transfer.
 payment.dashboard.header2= How to make a payment
-payment.dashboard.message2= Payments can be made by bank transfer.
-payment.dashboard.message3= To make a bank transfer payment, you can visit your bank’s website, use their mobile app, or go to a branch and use HMRC’s banking details.
+payment.dashboard.message2= To make a bank transfer, you can visit your bank''s website, use their mobile app, call the number at the back of your card or go to a branch. You will need to provide your bank with HMRC’s bank details.
```
Comment (jassalrichy) [inline conf/messages.en:178]: Incorrect apostrophe on line 178

File: conf/messages.en [hunk 8]
```diff
@@ -173,24 +173,27 @@ grs.registration.error.b33 =  and try again using different details if you think
...
+payment.dashboard.message3= Faster Payments will usually reach HMRC the same or next day, including weekends and bank holidays.
+payment.dashboard.message4= CHAPS payments usually reach HMRC the same working day if you pay within your bank’s processing times.
+payment.dashboard.message5= Bacs payments usually take 3 working days.
 payment.dashboard.timeline-summary= How long it takes
 payment.dashboard.timeline-content1= Faster Payments will usually reach HMRC the same or next day, including weekends and bank holidays.
-payment.dashboard.timeline-content2= CHAPS payments usually reach HMRC the same working day if you pay within your bank’s processing times.
-payment.dashboard.timeline-content3= Bacs payments usually take 3 working days.
-payment.dashboard.site.warning= Use {0} as your payment reference. HMRC will use this to match your payment.
-payment.dashboard.warning.message = You must use {0} as your payment reference. HMRC will use this to match your payment.
+payment.dashboard.site.warning= You must use {0} as your payment reference. We need this to match your payment.
```
Comment (jassalrichy) [inline conf/messages.en:185]: Is there a need to have two seprate keys for the same message?
payment.dashboard.site.warning 
payment.dashboard.warning.message

Comment (javed-iqbal1) [reply conf/messages.en:185]: one is for fallback for icon, one is for message, they may want change fallback message later for icon. it was used previously like this.

File: conf/messages.en [hunk 9]
```diff
@@ -192,7 +191,7 @@ payment.dashboard.table.row_12= Sort code
 payment.dashboard.table.row_22= Account number
 payment.dashboard.table.row_32= Account name
 payment.dashboard.table.caption= HMRC’s bank details for UK payments
-
+site.warning = Warning
```
Comment (jassalrichy) [inline conf/messages.en:194]: can we place site.warning at the top of the message file with all the other site keys please

**Review by jassalrichy — APPROVED** [review]

**Review by jassalrichy — DISMISSED** [review]

**Review by jassalrichy — APPROVED** [review]

**Pull request merged**